cmd/gcp-controller-manager/csr_signer.go

```diff
@@ -44,17 +43,23 @@ var (
 // ClusterSigningGKERetryBackoff is the backoff between GKE cluster signing retries.
 const ClusterSigningGKERetryBackoff = 500 * time.Millisecond
 
+type csrClient interface {
```

Comment:
Why not just use v1.CertificateSigningRequestInterface?

Reply:
Habit, I guess.  Is there an advantage either way?

Reply:
Moot now that I switched back to the single overall client at Jordan's suggestion.